src/foam/dashboard/view/Card.js

```diff
@@ -53,20 +53,26 @@ foam.CLASS({
     }
   ],
   css: `
-^ {
-  border: 2px solid #dae1e9;
-  border-radius: 2px;
-  background: white;
-}
+    ^ {
+      border: 2px solid #dae1e9;
+      border-radius: 2px;
+      background: white;
+      margin: 8px;
+      padding-bottom: 10px;
+    }
 
-^header {
```

Comment:
Update the forumla in the contentHeight/contentWidth expression to match how much width/height is left for the graph after your CSS changes.